CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to virtio-bindings
+
+## Dependencies
+
+### Bindgen
+The bindings are currently generated using
+[bindgen](https://crates.io/crates/bindgen) version 0.49.0:
+```bash
+cargo install bindgen --vers 0.49.0
+```
+
+### Linux Kernel
+Generating bindings depends on the Linux kernel, so you need to have the
+repository on your machine:
+
+```bash
+git clone https://github.com/torvalds/linux.git
+```
+
+## Example for adding a new version
+
+For this example we assume that you have both linux and virtio-bindings
+repositories in your root.
+
+```bash
+# Step 1: Crate a new module using a name with format "bindings_vVERSION" in
+# src/
+cd virtio-bindings
+mkdir src/bindings_v5_0_0
+cd ~
+
+# Step 2: Copy the "mod.rs" file from the directory of an already existing
+# version module to the one we've just created.
+cd virtio-bindings/src
+cp bindings_v4_14_0/mod.rs bindings_v5_0_0/mod.rs
+
+# linux is the repository that you cloned at the previous step.
+cd linux
+# Step 3: Checkout the version you want to generate the bindings for.
+git checkout v5.0
+
+# Step 4: Generate the bindings from the kernel headers. We need to
+# generate a file for each one of the virtio headers we're interested on.
+# For the moment, we're generating "virtio_blk", "virtio_net" and
+# "virtio_ring". Feel free to add additional header files if you need them
+# for your project.
+make headers_install INSTALL_HDR_PATH=v5_0_headers
+cd v5_0_headers
+for i in virtio_blk virtio_net virtio_ring ; do \
+    bindgen include/linux/$i.h -o $i.rs \
+    --with-derive-default \
+    --with-derive-partialeq \
+    --no-derive-debug
```

Comment:
Just wondering, why the `--no-derive-debug`?

Reply:
I don't think that's necessary indeed. I fixed it and made it consistent with the e.g. the KVM bindings crate.

Reply:
Cool! I imagine it's a bit of a bother, but if you are ok with having `Debug` implementations automatically generated it looks like `bindgen` should be run again. Everything looks good otherwise!

Reply:
Yes, they needed to be regenerated.
That's done now.